### PR TITLE
CON-1761: Microsoft graph identity protection app

### PR DESCRIPTION
**Pull request by milan-sumo**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+
+<img src={useBaseUrl('img/send-data/ms-graph.svg')} alt="dropbox-icon.png" width="50"/>
+
+The Microsoft Graph Identity Protection App for Sumo Logic provides organizations with advanced security monitoring and analytics capabilities for their Microsoft Graph Identity Protection data. The app leverages Sumo Logic's cloud-to-cloud source to collect and analyze data from the Microsoft Graph Identity Protection Source, allowing organizations to proactively identify potential security risks and take action to mitigate them.
+
+With the Microsoft Graph Identity Protection App, organizations can easily monitor and analyze events related to risky sign-ins, unusual user behavior, and security alerts generated by the Identity Protection API. The app provides a comprehensive view of these events, allowing security teams to quickly identify potential security incidents and take action to protect their organization.
+
+Key features of the Microsoft Graph Identity Protection App include:
+
+* Real-time monitoring: The app provides real-time monitoring of events related to the Microsoft Graph Identity Protection Source, allowing organizations to quickly detect potential security incidents.
```
Comment (stacykor) [inline docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md:18]:
```suggestion
* **Real-time monitoring.** The app provides real-time monitoring of events related to the Microsoft Graph Identity Protection Source, allowing you to quickly detect potential security incidents.
```

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+* Advanced analytics: The app includes a wide range of pre-built dashboards, and visualizations, that enable organizations to analyze their Microsoft Graph Identity Protection data in real-time, and gain insights into user behavior, sign-in patterns, and .risky detections. The App displays the distribution of risks by level, state, processing status, type, activity, etc which allows you to categorize risks by various criteria and helps to identify critical and frequently occurring risks.
```
Comment (stacykor) [inline docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md:19]:
```suggestion
* **Advanced analytics.** The app includes a wide range of pre-built dashboards, and visualizations, that enable organizations to analyze their Microsoft Graph Identity Protection data in real-time, and gain insights into user behavior, sign-in patterns, and .risky detections. The App displays the distribution of risks by level, state, processing status, type, activity, etc which allows you to categorize risks by various criteria and helps to identify critical and frequently occurring risks.
```

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+* Investigation capabilities: The app provides advanced search and analytics capabilities that allow security teams to investigate security incidents, identify the root cause of security events, and take steps to improve their organization's overall security posture.
```
Comment (stacykor) [inline docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md:20]:
```suggestion
* **Investigation capabilities.** The app provides advanced search and analytics capabilities that allow security teams to investigate security incidents, identify the root cause of security events, and take steps to improve their organization's overall security posture.
```

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+| where risk_action matches "{{risk_action}}"
+| where location matches "{{location}}"
+
+| count(risk_id) by risk_level
+| sort by _count
+```
+
+## Collecting Logs for Microsoft Graph Identity Protection API
+
+This section explains how to collect logs from Microsoft Graph Identity Protection API and ingest them into Sumo Logic. Refer to the [Cloud-to-Cloud-Integration for Microsoft Graph Identity Protection Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/ms-graph-identity-protection-source/) to create the source and use the same source category while installing the app.
```
Comment (stacykor) [inline docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md:128]:
```suggestion
This section explains how to collect logs from Microsoft Graph Identity Protection API and ingest them into Sumo Logic. For details, see [Cloud-to-Cloud-Integration for Microsoft Graph Identity Protection Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/ms-graph-identity-protection-source/) to create the source and use the same source category while installing the app.
```

**Review by stacykor — CHANGES_REQUESTED** [review]
Minor style stuff.

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md (hunk 1 above)
Comment (milan-sumo) [reply docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md]: Done

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md (hunk 2 above)
Comment (milan-sumo) [reply docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md]: Done

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md (hunk 3 above)
Comment (milan-sumo) [reply docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md]: Done

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md (hunk 4 above)
Comment (milan-sumo) [reply docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md]: Done

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+
+The App uses Microsoft Graph Identity Protection Source to collect [Risky Users](https://docs.microsoft.com/en-us/graph/api/riskyuser-list?view=graph-rest-1.0) and [Risk Detections](https://docs.microsoft.com/en-us/graph/api/riskdetection-list?view=graph-rest-1.0) data from Microsoft Graph Identity Protection API. 
+
+To learn about the Microsoft Graph Identity Protection API fields, see the [Risky Users](https://learn.microsoft.com/en-us/graph/api/resources/riskyuser?view=graph-rest-1.0#properties) and [Risk Detections](https://learn.microsoft.com/en-us/graph/api/resources/riskdetection?view=graph-rest-1.0#properties) documentation. 
+
+### Sample Log Messages
+
+This section contains log messages for both `Risky Users` and `Risk detections` sources.
+
+**Sample Risky Users Log Message**
```
Comment (kimsauce) [inline docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md:32]: Redundant

```suggestion
**Sample Risky Users Log Message**
```

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md [hunk 6]
```diff
@@ -0,0 +1,173 @@
...
+   "riskLevel": "high",
+   "riskState": "atRisk",
+   "riskDetail": "none",
+   "riskLastUpdatedDateTime": "2022-07-08T20:07:11.9432076Z",
+   "userDisplayName": "hitarth",
+   "userPrincipalName": "hitarth@sumo.onmicrosoft.com"
+}
+```
+
+**Sample Risk detections Log Message**
```
Comment (kimsauce) [inline docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md:48]: Redundant

```suggestion
```

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md [hunk 7]
```diff
@@ -0,0 +1,173 @@
+---
+id: microsoft-graph-identity-protection
+title: Microsoft Graph Identity Protection
+sidebar_label: Microsoft Graph Identity Protection
+description: This Sumo Logic App for Microsoft Graph Identity Protection provides organizations with advanced security monitoring and analytics capabilities for their Microsoft Graph Identity Protection data.
```
Comment (kimsauce) [inline docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md:5]:
```suggestion
description: The Sumo Logic App for Microsoft Graph Identity Protection provides organizations with advanced security monitoring and analytics capabilities for their Microsoft Graph Identity Protection data.
```

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md [hunk 8]
```diff
@@ -0,0 +1,173 @@
...
+
+Locate and install the app from the **App Catalog**. If you want to see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
+
+To install the app, follow the steps below:
+1. From the **App Catalog**, search for the app and select it.
+2. Select the version of the service you're using and click **Add to Library**.
+
+:::note
+Version selection is applicable only to a few apps currently. For details, see the [Install the Apps from the Library](/docs/get-started/apps-integrations#install-apps-from-the-library).
+:::
```
Comment (kimsauce) [inline docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md:140]:
```suggestion
2. Select the version of the service you're using and click **Add to Library**. Version selection is available only for limited number of apps. For details, see the [Install the Apps from the Library](/docs/get-started/apps-integrations#install-apps-from-the-library).
```

File: docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md [hunk 9]
```diff
@@ -0,0 +1,173 @@
...
+
+The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.
+
+## Viewing Microsoft Graph Identity Protection Dashboards
+
+* All dashboard have a set of filters that you can apply to the entire dashboard, as shown in the following example. Click the funnel icon in the top dashboard menu bar to display a scrollable list of filters that are applied across the entire dashboard.
+* You can use filters to drill down and examine the data on a granular level. Filters include client country, client device type, client IP, client request host, client request URI, client request user agent, edge response status, origin IP, and origin response status.
+* Each panel has a set of filters that are applied to the results for that panel only, as shown in the following example. Click the funnel icon in the top panel menu bar to display a list of panel-specific filters.
+
+###  Risk Detections
```
Comment (kimsauce) [inline docs/integrations/microsoft-azure/microsoft-graph-identity-protection.md:159]:
```suggestion
### Risk Detections
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
minor changes

**Review by kimsauce — CHANGES_REQUESTED** [review]
style stuff

**Review by kimsauce — APPROVED** [review]

**Pull request merged**